### PR TITLE
AV-173914 fix controller version to 22.1.2 for version >= 22.1.3

### PR DESCRIPTION
**Pull request by arihantg**

### CODE REVIEW

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
result of test_insecure_ingress_evh
image

**Comment by arihantg** [comment]
test_insecure_ingress_evh results
image

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -46,8 +46,8 @@ var runtimeScheme = k8sruntime.NewScheme()
 func init() {
 	//Setting the package-wide version
 	CtrlVersion = os.Getenv("CTRL_VERSION")
-	// For AKO 1.9.1, if ctrl version is 22.1.3, make it 22.1.2
-	if CtrlVersion == CTRL_VERSION_22_1_3 {
+	// For AKO 1.9.x, if ctrl version is >= 22.1.3, make it 22.1.2
+	if CtrlVersion >= CTRL_VERSION_22_1_3 {
```
Comment (akshayhavile) [inline pkg/utils/utils.go:50]: @arihantg: Similar change has to be done for: https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/release-1.9.3/pkg/utils/avi_rest_utils.go#L104

Comment (arihantg) [reply pkg/utils/utils.go:50]: Update there as well

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**